documentation/modules/ref-address-space-example-exposing-endpoints.adoc

```diff
@@ -38,16 +37,16 @@ spec:
     - 10.0.0.0/8
 ----
 <1> (Required) The name of the endpoint. The name specified affects the name of the {KubePlatform} service to be created as well as the name of the endpoint in the status section of the `AddressSpace`.
-<2> (Required) The service names that must be configured for the `AddressSpace`.  The service can either be `messaging`, `console`, or `mqtt`. However, the `mqtt` service is supported for the `standard` address space type only.
-<3> (Required) The type of endpoint being exposed. The `loadbalancer` type will create an {Kubernetes} `LoadBalancer` service. Valid values are `route` and `loadbalancer`.
-<4> (Required) A list of the ports to be exposed on the `LoadBalancer` service. For `messaging`, `amqp` and `amqps` can be specified.
+<2> (Required) The service names that must be configured for the `AddressSpace`.  Valid values are `messaging`, `console`, and `mqtt`. However, the `mqtt` service is supported for the `standard` address space type only.
```

Comment:
is this correct, there are services and service names - are they not distinct concepts, the way this is rewritten it makes the *names* only have certain valid values rather than the value for service